timemachine/md/moves.py

```diff
@@ -73,16 +74,16 @@ def n_proposed(self):
 class NPTMove(MonteCarloMove):
     def __init__(
         self,
-        ubps,
-        lamb,
-        masses,
-        temperature,
-        pressure,
-        n_steps,
-        seed,
-        dt=1.5e-3,
-        friction=1.0,
-        barostat_interval=5,
+        ubps: List,
```

Comment:
nit: use more specific `List[CustomOpWrapper]`?

Not the fault of this PR, but I think `ubps` is also a misnomer?

Reply:
note: I think we'd have to add `assert isinstance(ubps[0], HarmonicBond)` before `bond_list = get_bond_list(ubps[0])` below for this to work.

Reply:
`ubps` is definitely unclear, not sure what to call it though and we use `ubps` for these binded (having called `.bind()`)  potentials in a lot of places. 

Adding in `List[CustomOpWrapper]` causes ln 93 to fail as it expects a `HarmonicBond` type. Will see if there is a way around this

Reply:
> Will see if there is a way around this

I think my comment above should work

> add assert isinstance(ubps[0], HarmonicBond) before bond_list = get_bond_list(ubps[0])

Reply:
> ubps is definitely unclear, not sure what to call it though and we use ubps for these binded

Ah, that's true; I think my comment about it being a misnomer was a bit confused; this does make sense to me now to contrast with bound potentials

Reply:
Resolved in [a1d2bbf](https://github.com/proteneer/timemachine/pull/724/commits/a1d2bbf1be0d77f4282149c66ff3a743e193fd25)

Reply:
Ah, nice, I hadn't used `cast` before. That seems nice for some situations in that it's apparently completely ignored at runtime. In this case though, I'm on the fence -- it seems like the assertion might be nice to have anyway so there's a clear error message if the user passes in a list of potentials where `ubp[0]` is not a `HarmonicBond`?

Reply:
Will add in the assertion

Reply:
Resolved in [8c394ab](https://github.com/proteneer/timemachine/pull/724/commits/8c394ab189e4d3cfdddf1420555d8242bc24d609)